docs/tracking.md

```diff
@@ -31,40 +31,70 @@ please open an issue on [this issue tracker](https://github.com/Quansight-Labs/f
+| orjson               |         [:simple-github:](https://github.com/ijl/orjson/issues/530)          |                       |                            |                            |                                                            |                                                                                          |
+| packaging            |       [:simple-github:](https://github.com/pypa/packaging/issues/727)        | :material-check-bold: |   :material-check-bold:    |            24.0            |                                                            |                                                                                          |
+| pandas               |     [:simple-github:](https://github.com/pandas-dev/pandas/issues/59057)     | :material-check-bold: |   :material-check-bold:    |           2.2.3            | :simple-linux: :simple-apple: :material-microsoft-windows: |                                                                                          |
+| Pillow               |    [:simple-github:](https://github.com/python-pillow/Pillow/issues/8199)    | :material-check-bold: |   :material-check-bold:    |           11.0.0           | :simple-linux: :simple-apple: :material-microsoft-windows: |    [:simple-anaconda:](https://anaconda.org/scientific-python-nightly-wheels/pillow/)    |
+| pip                  |         [:simple-github:](https://github.com/pypa/pip/issues/12634)          | :material-check-bold: |   :material-check-bold:    |            24.1            |                                                            |                                                                                          |
+| PyArrow              |       [:simple-github:](https://github.com/apache/arrow/issues/43536)        | :material-check-bold: |                            |           18.0.0           |               :simple-linux: :simple-apple:                |   [:simple-anaconda:](https://anaconda.org/scientific-python-nightly-wheels/pyarrow/)    |
+| pybind11             |      [:simple-github:](https://github.com/pybind/pybind11/issues/5112)       | :material-check-bold: |   :material-check-bold:    |            2.13            |                                                            |                                                                                          |
+| PyO3                 |         [:simple-github:](https://github.com/PyO3/pyo3/issues/4265)          | :material-check-bold: |                            |            0.23            |                                                            |                                                                                          |
```

Comment:
A bunch of duplicates crept into this table, PyO3 and other projects now have 2 entries. Can you redo? 

Easier to review without the huge reformat as well, pre-commit is actively unhelpful especially if it's not in a separate commit.